test/module/child_process/basics.js

```diff
@@ -0,0 +1,30 @@
+// Test basic child process functionality
+//
+
+async_test(function(){
+    var text = "hello";
+    var process = require('child_process');
+    var p = process.spawn("echo", ["-n", text]);
```

Comment:
Closest thing I'm aware of is this: http://winaero.com/blog/how-to-make-the-echo-command-without-new-line-in-windows/